doc/mclag/MCLAG_Enhancements_HLD.md

```diff
@@ -0,0 +1,808 @@
+3. Verify that keep-alive and session-timeout values are updated properly in CONFIG_DB table.
+4. Verify that MAC address updates received from peer MCLAG nodes is updated to APP MCLAG FDB table.
+5. Verify that ICCP session state information is updated to STATE_MCLAG_TABLE.
+6. Verify that MCLAG interface state information is updated to STATE_MCLAG_REMOTE_INTF_TABLE
+7. Verify that Unique IP configuration if add/delete is updated in CONFIG_DB MCLAG UNIQUEIP TABLE
+
+## 9.2 FDB
+
+1. Verify that MAC address learned locally on orphan port are synced to the peer MCLAG node. 
+2. Verify that MAC address aged locally on orphan ports are withdrawn to peer MCLAG node.
+3. Verify that MAC address learned locally on MCLAG portchannel are synced to the peer MCLAG node. 
+4. Verify that MAC address aged locally on MCLAG portchannel ports are withdrawn to peer MCLAG node.
+5. Verify that when MCLAG portchannel is down MAC addresses pointing to the MCLAG portchannel are updated to point to peer-link interface.
+6. Verify that when MClag portchannel is up MAC addresses pointing to peer-link interface are updated to point to MCLAG portchannel.
+7. Verify that when peer-link is down MAC addresses pointing to peer-link are deleted on both the MCLAG peer nodes.
+8. Verify that when clear FDB is executed local MAC addresses are cleared locally and withdrawn to the peer MCLAG node. Relearn the MAC addresses and advertise to peer MCLAG node.
+9. Verify that static MAC configured locally is advertised to peer MCLAG node.
+10. Verify that static MAC deleted locally is withdrawn to peer MCLAG node. 
+11. Verify that in presence of remote static MAC addresses any dynamic local MAC move is not allowed.
+12. Verify MAC learn with scale of 40K MAC entries.  
+
+## 9.3 Config handling
+
+1. Verify MCLAG domain configuration with local, peer ip, peer interface, MCLAG interface configures ICCP
+2. Verify adding MCLAG interface to existing MCLAG domain  
+3. Verify deleting one MCLAG interface from existing MCLAG domain  
+4. Verify modifying local ip configuration for an existing MCLAG domain
+5. Verify modifying peer interface for given MCLAG domain 
+6. Verify configuration change of MCLAG keep-alive timer value to min/max values are reflected
+7. Verify configuration change of MCLAG session timeout value to min/max values are reflected
+8. Verify deletion of existing MCLAG domain
+9. Verify adding MCLAG domain without peer interface
+10. Verify adding MCLAG peer interface later after adding MCLAG domain
+11. Verify deleting MCLAG interfaces with synced up MAC addresses and make sure behavior is similar to having MCLAG interface down
+12. Verify Unique IP configuration on VLAN interface
+
+## 9.4 Port Isolation
+
+1. Configure Port Isolation group with members apply it to ingress port. Send unicast and BUM traffic and verify traffic is not seen at any member ports.
```

Comment:
Could you please point to the example configs for Port Isolation groups?

Reply:
@prvattem: There is no explicit configuration required for Port Isolation group used for MCLAG peer_link. This gets internally created.